Lib/test/libregrtest/main.py

```diff
@@ -209,9 +213,6 @@ def find_tests(self, tests: TestList | None = None) -> tuple[TestTuple, TestList
                 sys.exit(1)
 
         if self.randomize:
-            if self.random_seed is None:
-                self.random_seed = random.randrange(100_000_000)
-            random.seed(self.random_seed)
```

Comment:
I'm not sure that removing seed() call is correct here. The goal is to be reproducible, but setup_tests() which calls random.seed() is not done in main.py, but later, when running tests, single.py.

Please restore the seed() call to make sure that shuffle() is always the same, even if we add random calls before shuffle().